src/components/Modals/NewMessageModal.jsx

```diff
@@ -129,103 +138,106 @@ const NewMessageModal = ({ showModal, setShowModal, oldMessage = '', toField = '
-          />
-          <TextField
-            margin="normal"
-            value={message.title}
-            type="text"
-            name="title"
-            id="title"
-            onChange={(e) => handleChange(e)}
-            required
-            label="Subject"
-            inputProps={{ maxLength: '48' }}
-            fullWidth
-          />
-          <TextField
-            margin="normal"
-            value={oldMessage && originalMessage}
-            type="text"
-            name="previousMessage"
-            id="previousMessage"
-            label="Previous Message"
-            variant="filled"
-            // TODO: The line below shrinks the "Reply To" version more than the "New Message" one
-            // Is this something that needs to be addressed?
-            sx={{ display: oldMessage ? 'block' : 'none' }}
-            multiline
-            rows={3}
-            InputProps={{
-              readOnly: true
-            }}
-            InputLabelProps={{
-              shrink: true
-            }}
-            fullWidth
-          />
-          <TextField
-            margin="normal"
-            value={message.message}
-            type="text"
-            name="message"
-            id="message"
-            onChange={(e) => handleChange(e)}
-            multiline
-            rows={5}
-            label="Enter Message"
-            required
-            // TODO: Determine how long a maximum length, if any, is suitable
-            inputProps={{ maxLength: '500' }}
-            fullWidth
-          />
-          <DialogActions sx={{ width: '100%' }}>
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: isSmallScreen ? 'column' : 'row',
-                gap: isSmallScreen ? '10px' : '8px',
-                justifyContent: 'center',
-                alignItems: 'center',
-                width: '100%'
+        <FormSection title={oldMessage ? 'Reply To' : 'New Message'} id="new-message-form">
+          <form
+            aria-labelledby="new-message-form"
+            onSubmit={(e) => handleSubmit(e)}
+            autoComplete="off"
+          >
+            <TextField
+              margin="normal"
+              value={toField || message.recipientPodUrl}
+              type="text"
+              name="recipientPodUrl"
+              id="recipientPodUrl"
+              onChange={(e) => handleChange(e)}
+              required
+              autoFocus
+              label="To"
+              fullWidth
+              disabled={toField !== ''}
+            />
+            <TextField
+              margin="normal"
+              value={message.title}
+              type="text"
+              name="title"
+              id="title"
+              onChange={(e) => handleChange(e)}
+              required
+              label="Subject"
+              inputProps={{ maxLength: '48' }}
+              fullWidth
+            />
+            <TextField
```

Comment:
This text field should be conditionally rendered. Even on a New Message modal, it still shows in the DOM currently.  Maybe `oldMessage && <TextField>`?